helm/values.yaml

```diff
@@ -105,18 +81,19 @@ idxworker:
   pool_size: 5
   debug: "FALSE"
   d1_serviceType_url: https://cn.dataone.org/mnServiceTypes.xml
+  tripleDbDirectory: /etc/dataone/tdb-cache
```

Comment:
What is this tdb-cache I am seeing throughout? I don't remember needing this before. I'll poke through the issues looking for some context.

Reply:
@taojing2002 can probably give more background, but basically it's used for temp storage when indexing a dataset. I noticed some log errors that we tracked down to being a permissions issue (the app was trying to write to the pod's filesystem unsuccessfully).